frontends/web/src/routes/account/add/add.tsx

```diff
@@ -109,29 +100,71 @@ class ManageAccount extends Component<Props, State> {
                 apiPost('account-add', {
                     coinCode,
                     name: accountName,
-                }).then((data: ResponseData) => {
-                    if (data.success) {
-                        this.setState({ step: 2 });
-                        //route('/account/' + data.accountCode);
-                    } else {
-                        if (data.errorCode) {
-                            alertUser(this.props.t(`addAccount.error.${data.errorCode}`));
-                        } else if (data.errorMessage) {
-                            alertUser(this.props.t('unknownError', { errorMessage: data.errorMessage }));
+                })
+                    .then((data: ResponseData) => {
+                        if (data.success) {
+                            this.setState({ step: 'success' });
+                            //route('/account/' + data.accountCode);
```

Comment:
cherry-pick these changes into this PR, or merge the other one then rebase this one?

https://github.com/digitalbitbox/bitbox-wallet-app/pull/1279

Reply:
cherry-picked